aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -99,11 +99,10 @@ class QConv2dInt8 final : public c10::OperatorKernel {
     PackedConvWeight& pack_ptr =
         cpp_custom_type_hack::cast<PackedConvWeight>(packed_weight);
     auto packB = pack_ptr.w.get();
-    // packB->printPackedMatrix("PackedB inside QConv2dInt8:");
```

Comment:
Still keep this for debugging purpose? Do we have printPackedMatrix function in the general Conv interface?

Reply:
We don't. Let me remove this.